### DISCUSSION TITLE
kong cant set target healthy

### DISCUSSION THREAD

**Discussion by alexDango** [Kong/kong#8219] [Help]
does anyone can help me about this.
Kong Enterprise 2.6.0.2

1 control-plane  2 data-plane 
Ring-balancer

curl http://localhost:8001/upstreams/demo_upstrem/health
{"next":null,"node_id":"f87e336a-387e-4166-9770-9f691e9838ea","data":[{"created_at":1639995462.118,"id":"e0854822-485e-4b3a-b79e-79b2cd57ffcb","weight":100,"target":"172.15.2.218:5959","tags":null,"upstream":{"id":"3514f199-6b4c-42b6-8f85-a403a63505bc"}}]}

curl http://localhost:8001/upstreams/demo_upstrem/targets/all
{"next":null,"data":[{"created_at":1639995462.118,"id":"e0854822-485e-4b3a-b79e-79b2cd57ffcb","weight":100,"target":"172.15.2.218:5959","tags":null,"upstream":{"id":"3514f199-6b4c-42b6-8f85-a403a63505bc"}}]}

curl http://localhost:8001/upstreams/demo_upstrem/targets/e0854822-485e-4b3a-b79e-79b2cd57ffcb/healthy -XPOST
{"message":"Upstream demo_upstrem has no balancer"}

**Comment by kikito** [comment]
That is strange.

Does the target still appear in http://localhost:8001/upstreams/demo_upstrem/targets/all after you try that failed POST?

Was this Kong started from scratch, or have you done some experimentation over it?

**Reply by alexDango** [reply]
still appear.  why the message is Upstream demo_upstrem has no balancer
